migrations/env.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import with_statement
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+from logging.config import fileConfig
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
```

Comment:
I think we should remove all this comments. They are just polluting the code.

Reply:
I agree, but I've made a few notes to create an issue just to make a clean up, adjust the code style to be in line with flake8 and normalise the comments to follow google style for documentation.

Reply:
Thumbs up for following flake8 && google style. I'm ok with leaving these comments clean up to another PR.